src/AppService/readme.md

```diff
@@ -323,4 +323,8 @@ directive:
   - where:
       variant: (.*)ViaIdentitySlot
     remove: true
+  # Make this a preview module
+  - from: source-file-csharp
+    where: $
+    transform: $ = $.replace('sb.AppendLine\(\$@\"\{Indent\}\{Indent\}\{Indent\}ReleaseNotes = \'\'\"\);', 'sb.AppendLine\(\$@\"\{Indent\}\{Indent\}\{Indent\}ReleaseNotes = \'\'\"\);\n            sb.AppendLine\(\$@\"\{Indent\}\{Indent\}\{Indent\}Prerelease = \'preview\'\"\);' );
```

Comment:
Repeated `sb.AppendLine\(\$@\"\{Indent\}\{Indent\}\{Indent\}ReleaseNotes = \'\'\"\);`, is it on purpose?

Reply:
Yes, replacing the existing AppendLine with itself +  a new AppandLine